File: Learning c & c++/c/practice12.c
// The use of non-constant expressions in case labels is not allowed in C.
#include <stdio.h>

int main() {
    int num1, num2, num3;

    printf("Enter three numbers: ");
    scanf("%d %d %d", &num1, &num2, &num3);

    printf("Comparison Results:\n");

    switch (1) {
        case (num1 > num2 && num1 > num3):
            printf("%d is the largest number.\n", num1);
            break;
        case (num2 > num1 && num2 > num3):
            printf("%d is the largest number.\n", num2);
            break;
        case (num3 > num1 && num3 > num2):
            printf("%d is the largest number.\n", num3);
            break;
        default:
            printf("There is no distinct largest number; at least two numbers are equal.\n");
    }

    return 0;
}

File: Python/practice/practice12.py
# max between three numbers
print('Enter three numbers.')
a = int(input('a='))
b = int(input('b='))
c = int(input('c='))
# if a > b:
#     if a > c:
#         print(a, 'is the greatest number.')
#     else:
#         print(c, 'is the greatest number.')
# elif b > c:
#     print(b, 'is the greatest number.')
# else:
#     print(c, 'is the greatest number.')
if a > b and a > c:
    print(a, 'is the greatest number.')
elif a > b or b <= c:
    print(c, 'is the greatest number.')
else:
    print(b, 'is the greatest number.')
# if a>b and a>c:
#     max=a
# elif b>a and b>c:
#     max=b
# else:
#     max=c
# print(max,'is the greatest number.')
